internal/metrics/metrics.go

```diff
@@ -11,6 +11,13 @@ var StartTimeGauge = promauto.NewGauge(prometheus.GaugeOpts{
 	Help: "The timestamp of the time when the app started",
 })
 
+// GraphQueryTimeExecution reports the time execution in ms for queries.
+var GraphQueryTimeExecution = promauto.NewHistogramVec(prometheus.HistogramOpts{
+	Name:    "go_graphkb_graph_query_execution_time",
+	Help:    "The time execution in ms of queries.",
+	Buckets: []float64{1, 50, 75, 100, 500, 1000, 2500, 5000, 10000},
```

Comment:
Opt: use `prometheus.ExponentialBuckets()` to generate buckets

Reply:
I'll leave it like this for now as I'd like to see the frequency of the observations within these quantiles first.